GMINER/1.15/.nvoc-miner.json

```diff
@@ -1,13 +1,14 @@
 {
 	"friendlyname": "gminer",
 	"version": "1.15",
-	"hcd": "miner --server ${!xpool} --user ${!xaddr}${!xwallet}${!xwork} --port ${!xport} --pass $MINER_PWD --time $GMINER_OPTS ${!xext}",	"devlist_argument": "devices=",
```

Comment:
Why removed $GMINER_OPTS?

Reply:
its inside ${!xext}. Not sure if "--time" was needed instead.

Reply:
I dont think ${!xext} covers both COIN_EXTENSION_ARGUMENTS and MINER_OPTS, do you?
From 0miner:
```
UEXT="_EXTENSION_ARGUMENTS"
xext=$COIN$UEXT
```

Reply:
--time was not needed, mistaken with DSTM

Reply:
It should…
https://github.com/papampi/nvOC_by_fullzero_Community_Release/blob/93dc900251680375d1b0f1e1c5e2438d39f7f78e/0miner#L120

of course any legacy _extension_arguments is now _OPTS

Reply:
Oh, and actually ${!xext} has been renamed to just OPTS too, and its always appended from 0miner, so here we should  remove $!xext} as well.